src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/assets/baselines/Licenses.runtime.json

```diff
@@ -0,0 +1,52 @@
+{
+  "files": [
+    {
+      "path": "PATENTS.TXT",
+      "detected_license_expression": "ms-patent-promise"
```

Comment:
Sorry, I am little confused about this, but how do we decide what to add to a baseline vs what to list in the acceptable license list?

Reply:
Speaking of `ms-patent-promise`, I think we should add it to our allow-list. It's not exactly a license, but neither is the `newton-king-cla`, which is in our allow list.

(Aside: our lawyers have said that both `ms-patent-promise` and `newton-king-cla` are technically not licenses. They don't need to be a part of our license-scan results).

Reply:
Here's the rules I've been operating under:

1. If it's an allowed open-source license, add it to the list of allowed licenses in `LicenseScanTests.cs`.
1. If the file shouldn't be scanned as a general rule because of its file type, add it to the list of excluded file types in `LicenseScanTests.cs`.
1. Add it to `LicenseExclusions.txt` if the referenced license is one of the following:
  a. Not applicable (e.g. test data)
  b. False positive
1. Everything else should go in the baseline file.

I'll go ahead and add `ms-patent-promise` to the allowed list.

Reply:
Fixed in https://github.com/dotnet/installer/pull/17442/commits/f3972a6f44f09659fe412614a65f60aa65f38e3a